.github/workflows/dist-tag.yml

```diff
@@ -0,0 +1,33 @@
+name: dist-tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: Version to dist tag.
+        required: true
+      dist-tag:
+        type: string
+        description: Dist tag to use.
+        required: true
+
+jobs:
+  dist-tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: elastic/apm-pipeline-library/.github/actions/setup-npmrc@current
+        with:
+          secret: secret/jenkins-ci/npmjs/elasticmachine
+          vault-url: ${{ secrets.VAULT_ADDR }}
+          vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
+          vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+      - uses: ./.github/actions/setup-totp
+        with:
+          vault-url: ${{ secrets.VAULT_ADDR }}
+          vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
+          vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+      - run: |
```

Comment:
NIT: can you add a name for the step since it's a multiline step?

Reply:
https://github.com/elastic/synthetics/pull/744/commits/344dc02c6e96151d180ac095be9c5fb7d28d0da8